student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(_clasroom, age, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] There is a typo in "_clasroom" should be "classroom" to match with the instance variable. You don't need to add the underscore sign in this case.